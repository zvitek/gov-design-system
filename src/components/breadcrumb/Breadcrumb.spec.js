import { shallowMount } from '@vue/test-utils'
import GovBreadcrumb from '@components/breadcrumb/Breadcrumb'

describe('GovBreadcrumb', () => {
    let wrapper

    beforeEach(() => {
        wrapper = shallowMount(GovBreadcrumb)
    })

    it('is called', () => {
        expect(wrapper.name()).toBe('GovBreadcrumb')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
