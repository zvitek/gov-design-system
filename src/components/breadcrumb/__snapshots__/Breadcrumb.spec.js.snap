// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GovBreadcrumb render correctly 1`] = `<nav class="gov-breadcrumbs"></nav>`;
